### DISCUSSION TITLE
How to run SeisSol_proxy_Release_dhsw_6_elastic and what it can calculate

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#571] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/104721358/172313768-cbcc0608-cf50-4ee7-b941-399caaa4409f.png)

**Comment by sebwolf-de** [comment, marked as answer]
Hi,
this is our performance proxy for single-node execution. It executes the SeisSol kernels on random data. This way, you will get an estimate how fast you can run a SeisSol on your cluster. Execute `./SeisSol_proxy_Release_dhsw_6_elastic 100000 10 all` for a first overview.

**Reply by smallriver666** [reply]
> Hi, this is our performance proxy for single-node execution. It executes the SeisSol kernels on random data. This way, you will get an estimate how fast you can run a SeisSol on your cluster. Execute `./SeisSol_proxy_Release_dhsw_6_elastic 100000 10 all` for a first overview.

thank you very much，maybe it need long time to run Example tpv5 in my own computer?
This is my computer configuration:
![image](https://user-images.githubusercontent.com/104721358/172340325-11fceaa7-1a05-4b1a-ad27-39edfb46a6e6.png)

**Comment by smallriver666** [comment]
thank you very much，maybe it need long time to run Example tpv5 in my own computer?
This is my computer configuration:
![image](https://user-images.githubusercontent.com/104721358/172340325-11fceaa7-1a05-4b1a-ad27-39edfb46a6e6.png)

**Reply by sebwolf-de** [reply]
It is hard to judge the compute time on your computer. It seems kind of recent though. 
Enlarge these values to get a mesh with fewer elements to reduce the overall compute time.
https://github.com/SeisSol/Examples/blob/master/tpv5/tpv5_f200m.geo#L43:L44
